t/14-isotovideo.t

```diff
@@ -105,6 +105,20 @@ subtest 'isotovideo with git refspec specified' => sub {
             opts => "casedir=$data_dir/tests test_git_refspec=deadbeef _exit_after_schedule=1") } qr/Checking.*local.*deadbeef/, 'refspec in local git repository would be checked out';
 };
 
+subtest 'isotovideo with wheels' => sub {
+    chdir($pool_dir);
+    unlink('vars.json') if -e 'vars.json';
+
+    my $specfile = path("$data_dir/tests/wheels.yaml");
+    $specfile->spurt("wheels: [foo/bar]");
+    combined_like { isotovideo(
+            opts => "casedir=$data_dir/tests _exit_after_schedule=1") } qr@Invalid.*Missing property@, 'invalid YAML causes error';
+    $specfile->spurt("version: v99\nwheels: [foo/bar]");
+    combined_like { isotovideo(
+            opts => "casedir=$data_dir/tests _exit_after_schedule=1") } qr@Unsupported version@, 'unsupported version';
+    $specfile->remove;
```

Comment:
Please use the END block at the end of the test as we do in all our tests to make sure files are also removed in case the test exits unexpectedly.

Reply:
I removed it because it is executed too late in the successful case

Reply:
>  removed it because it is executed too late in the successful case
By too late I mean that other tests will pickup the file. Although we could have both I guess 🤔